rust/libnewsboat/src/utils.rs

```diff
@@ -87,6 +88,32 @@ pub fn absolute_url(base_url: &str, link: &str) -> String {
         .to_owned()
 }
 
+pub fn resolve_tilde(path: String) -> String {
+    let mut file_path:String = path;
+    let home_path = dirs::home_dir();
+
+    if home_path.is_some() {
+        let home_path_string = home_path.unwrap().display().to_string();
+
+        if file_path == "~" {
+            file_path = home_path_string;
+        }
+        else{
```

Comment:
Please put `else` on the same line with the curly braces, and use spaces around it: `} else {`